src/lib/functions/netlify-function.js

```diff
@@ -1,11 +1,20 @@
 // @ts-check
+const CronParser = require('cron-parser')
+
 const { error: errorExit } = require('../../utils/command-helpers')
 
 const BACKGROUND_SUFFIX = '-background'
 
 // Returns a new set with all elements of `setA` that don't exist in `setB`.
 const difference = (setA, setB) => new Set([...setA].filter((item) => !setB.has(item)))
 
+const getNextRun = function (schedule) {
```

Comment:
[dust] Any reason for to have a second `getNextRun` method outside of the class, as opposed to having this logic in `NetlifyFunction.prototype.getNextRun`?

Reply:
no specific reason 🤷 copied over from the other file, where there wasn't a prototype to place this on. also easier testable, potentially.